src/clr-angular/layout/nav/_responsive-nav.clarity.scss

```diff
@@ -340,6 +340,14 @@
     }
 
     .main-container {
+      .clr-example & {
```

Comment:
Oops, I missed reading the filename here, what we want is this CSS to go into the website somewhere, not into clr-angular package.

Reply:
Good catch! I've made the appropriate changes.